<ui version="4.0" >
 <class>CoverOptionsPage</class>
 <widget class="QWidget" name="CoverOptionsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>279</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="rename_files" >
     <property name="title" >
      <string>Location</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="save_images_to_tags" >
        <property name="text" >
         <string>Embed cover images into tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="save_images_to_files" >
        <property name="text" >
         <string>Save cover images as separate files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cover_image_filename" />
      </item>
      <item>
       <widget class="QCheckBox" name="save_images_overwrite" >
        <property name="text" >
         <string>Overwrite the file if it already exists</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>275</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>save_images_to_tags</tabstop>
  <tabstop>save_images_to_files</tabstop>
  <tabstop>cover_image_filename</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
